README.md

```diff
@@ -42,7 +42,7 @@ Please see the separate [sdk-packages](https://github.com/git-for-windows/sdk-pa
 
 ## Getting in contact
 
-We now have a [mailing list](https://groups.google.com/group/git-win-sdk) for developers.
+We now have a [mailing list](https://groups.google.com/group/git-win-sdk) for developers.  Issues should be reported on the [git-for-windows/git tracker](https://github.com/git-for-windows/git/issues); mentioning "SDK" in the subject line is advisable.
```

Comment:
Hmm, a few things:
- We usually do not use two spaces after ".", just one.
- I'd prefer if the name of the link would say "Git for Windows issue tracker".
- Actually, mentioning "SDK" in the subject is not the recommended way to note this is an SDK issue. Just label the issue with "sdk" instead.
- Please sign-off your commit.